packages/core/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt

```diff
@@ -119,13 +123,31 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
         }
 
         animationURL?.let {
-            view.setAnimationFromUrl(it, it.hashCode().toString())
+            if (it.startsWith('/') || it.startsWith("file://")) {
```

Comment:
Can you give me an example for starting with `/` ? I feel like this checking can be unreliable.

Reply:
I think `startsWith('/')` is should reliable for unix file system. We could use `File.exists()` instead, which should be cleaner.

Reply:
Yes! Much better :) Thanks.